libcontainer/specconv/spec_linux.go

```diff
@@ -190,6 +191,23 @@ func CreateLibcontainerConfig(opts *CreateOpts) (*configs.Config, error) {
 
 	exists := false
 	for _, m := range spec.Mounts {
+		// check mount options for (b)bind type mount
+		if m.Type == "bind" || m.Type == "rbind" {
+			if len(m.Options) == 0 {
+				logrus.Warnf("no options specified for %v type mount: [destination=%v, source=%v].", m.Type, m.Destination, m.Source)
```

Comment:
This warning should be the same as the other one. The code should be something like:

```
hasBindOpt := false
for _, opt := m.Options {
  // ...
}
if !hasBindOpt {
  logrus.Warnf(...)
}
```

In particular you don't need to special-case `len(m.Options) == 0`.

Reply:
Updated.
I still have a suggestion. How about use `fprintf(stderr, "....")` instead of `logrus.Warnf`?
Because user must use `--log /dev/stdout` to get the warning. And rootless container don't have access to `/dev/stdout`.